README.md

```diff
@@ -547,70 +547,213 @@ steps:
         skip: true # prevents scheduling the checkout container
 ```
 
-### Overriding flags for git clone/fetch
+### Overriding flags for git clone and git fetch (v0.13.0 and later)
 
-`git clone` and `git fetch` flags can be overridden per-step (similar to `BUILDKITE_GIT_CLONE_FLAGS` and `BUILDLKITE_GIT_FETCH_FLAGS` env vars) with the `checkout` block also:
+Flags for `git clone`, `git fetch` can be overridden per-step (similar to
+`BUILDKITE_GIT_CLONE_FLAGS` and `BUILDLKITE_GIT_FETCH_FLAGS` env vars) with
+the `checkout` block also:
 
 ```yaml
+# pipeline.yml
 steps:
 - label: Hello World!
   agents:
     queue: kubernetes
   plugins:
-  - kubernetes:
-      checkout:
-        cloneFlags: -v --depth 1
-        fetchFlags: -v --prune --tags
+    - kubernetes:
+        checkout:
+          cloneFlags: -v --depth 1
+          fetchFlags: -v --prune --tags
```

Comment:
Noting that the indent of the `steps` array and `plugins` array don't match with this change

Reply:
Fixed.